mk/config.mk

```diff
@@ -176,6 +176,26 @@ CFG_WITH_USER_TA ?= y
 # case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Support for loading user TAs from a special section in the TEE binary
+# Such TAs are available even before tee-supplicant is available (hence their
+# name), but note that many services exported to TAs may need tee-supplicant,
+# so early use is limited to a subset of the TEE Internal Core API (crypto...)
+# To use this feature, set EARLY_TA to the paths to one or more TA ELF file(s)
+# For example:
+#   EARLY_TA=path/to/8aaaf200-2450-11e4-abe2-0002a5d5c51b.stripped.elf
```

Comment:
I'll update the comment. What's expected is one or more paths to TA files separated with spaces, such as:
```
EARLY_TA="../optee_test/out/ta/crypt/cb3e5ba0-adf1-11e0-998b-0002a5d5c51b.stripped.elf ../optee_test/out/ta/aes_perf/e626662e-c0e2-485c-b8c8-09fbce6edf3d.stripped.elf"
```

Regarding the lack of `CFG_` prefix: you're mostly right but there's a trick (that I should document): all `CFG_` variables are written to `$(out-dir)/conf.mk` and any change to the values in this file causes a re-compilation of everything. So, consider the build steps I have given as an example:
```
$ make CFG_EARLY_TA=y            # (1)
<build some TAs>                 # (2)
$ make ... EARLY_TA=/some/path   # (3)
```
You can't give `CFG_EARLY_TA_PATH` during step (1) since we assume the TAs have not been built yet (they need optee_os to be built first so that dev_kit is available). So you can't have `CFG_EARLY_TA_PATH` set to the same value in steps (1) and (3) thus causing a full rebuilt of optee_os at step (3).

I propose to rename `EARLY_TA` and use `BUILTIN_TA_PATHS` instead (will do it at the same time I replace "early" with "builtin").